src/stylesheets/uswds.scss

```diff
@@ -48,6 +48,7 @@ Output all USWDS
 @import 'components/accordions';
 @import 'components/alerts';
 @import 'components/banner';
+@import 'components/checklist';
```

Comment:
Isn't a checklist an input like radio buttons?

Reply:
Hm, not in this case — this is a component (in v1 as well) called `.usa-checklist` which is essentially a validation checklist of "things you need to do". Definitely open to new names for it, though.

See:
https://components.designsystem.digital.gov/components/preview/validation.html
and
https://github.com/uswds/uswds/blob/e9249a0fb47733a065a35937eda06e5a47d00f22/src/stylesheets/components/_forms.scss#L113

Reply:
It felt like it _could_ be used in more cases than just in a forms alert, though, which is why I think it should be abstracted into its own component

Reply:
Oh, I thought this was checkbox and didn't see the connection with validation.